.changeset/chilly-moles-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@manypkg/tools": major
+---
+
+feat: serializable return values
```

Comment:
```suggestion
Return values became serializable again. An identifier for the detected tool is returned instead of an instance of a `Tool` object.
```